.github/workflows/cut-all-release-branches.yml

```diff
@@ -0,0 +1,43 @@
+# **what?**
```

Comment:
Is this the YAML file that each repo would need to have in it? A little confused why we need this vs just the other file below with all the implementation

Reply:
I've been going back and forth with how the process should be implemented.  

This would orchestrate cutting all the release branches at once.  Since it opens PRs for the changes to main, it's just a matter of merging in `dbt-core` and rerunning all the CI for the adapters before merging those in.  If we need to do just a single repo, we still can with `cut-release-branch.yml`.

Alternatively, we can run this manually in each repo.  Once this is proven, it could be added as an initial step for the release to run if a flag is set (or more simply if this is an rc1 release).

Reply:
Responding to you made me go back to running in each repo.  Deleting this workflow.